watch-parties/src/java/Room.java

```diff
@@ -0,0 +1,15 @@
+import java.util.ArrayList;
```

Comment:
Please remove this file from this PR. We should keep PRs focused on one thing. It's a good practice to use a new branch for different tasks (otherwise it's nigh-impossible to keep PRs focused without blocking yourself).